src/api/orders.rs

```diff
@@ -0,0 +1,119 @@
+use serde::{Deserialize, Serialize};
+
+use crate::{client::Client, error::Error};
+
+#[derive(Serialize, Deserialize, Debug)]
+// The struct for placing an order - body of the post request
+pub struct OrderPlacing {
+    pub isin: String,
+    pub expires_at: Option<String>,
+    pub side: OrderType, // "buy" or "sell"
+    pub quantity: i64,
+    pub venue: Option<String>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub enum OrderType {
+    buy,
+    sell
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+// The struct for placing an order - the response of the request
+pub struct OrderPlacingResponse<T> {
+    pub time: String,
+    pub status: String,
+    pub mode: String,
+    pub results: Option<T>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+// The struct for placing an order - the results response of the request
+pub struct OrderResults {
+    pub created_at: String,
+    pub id: String,
+    pub status: String,
+    pub regulatory_information: Option<RegulatoryInformation>,
+    pub isin: Option<String>,
+    pub expires_at: Option<String>,
```

Comment:
Same as the comment above regarding datetime type

Reply:
Well, the "expires_at" in the body of the request requires the format YYYY-MM-DD, do you know if chrono::Datetime have a stronger type for formatted date?

Reply:
Formatted date should be `NaiveDate`